stable/storageos-operator/app-readme.md

```diff
@@ -3,18 +3,25 @@
 [StorageOS](https://storageos.com) is a cloud native, software-defined storage
 platform that transforms commodity server or cloud based disk capacity into
 enterprise-class persistent storage for containers. StorageOS is ideal for
-deploying databases, message busses, and other mission-critical stateful
+deploying databases, message quesues, and other mission-critical stateful
```

Comment:
quesues -> queues

Reply:
I changed this recently - did I make the typo and did you copy it from somewhere else that needs to be corrected too?

Reply:
Yes, I copied all of the text changes from the helm3 PR.